BackofficeBundle/Resources/views/BackOffice/Underscore/table/tableHeader/searchField/tableBooleanField._tpl.twig

```diff
@@ -0,0 +1,10 @@
+<td data-column="<%=  column %>" rowspan="1" colspan="1">
+    <label class="select">
+        <select class="form-control search-column">
+            <option value=""></option>
+            <option value="1">True</option>
```

Comment:
This options are not translated beacause the values in the list are not translated. A ticket is created to translate the boolean value. so when the boolean value will be translated, it will also translate this options